spec/javascripts/angular/services/querySnapshotSvc_spec.js

```diff
@@ -225,7 +227,7 @@ describe('Service: querySnapshotSvc', function () {
     it('resolves ids->docs', function resolveAddSnapTest() {
       $httpBackend.expectPOST('/api/cases/2/snapshots')
                   .respond(200, addedSnapResp);
-      querySnapshotSvc.addSnapshot('myname', mockQueries);
+      querySnapshotSvc.addSnapshot('myname', recordDocumentFields, mockQueries);
       $httpBackend.flush();
```

Comment:
it might be good to add a test that recordDocumentFields works with some other setting besides false?